website/events/api/calendarjs/urls.py

```diff
@@ -25,4 +26,10 @@
         CalendarJSUnpublishedEventListView.as_view(),
         name="calendarjs-unpublished",
     ),
+    path(
+        "external/unpublished/",
```

Comment:
```suggestion
        "external-events/unpublished/",
```

Reply:
I simply followed the conventions set by the other endpoints, and there is no `external-events/` endpoint, only `external/`